core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1531,21 +1547,19 @@ public static boolean isClassProperty(String key) {
     propertiesByKey = new HashMap<>();
     validPrefixes = new HashSet<>();
     validProperties = new HashSet<>();
+    validTableProperties = new HashSet<>();
```

Comment:
These don't even need to be assigned here. These 4 lines could be deleted, if the `= new HashMap<>();` part were appended to their respective `private static final` lines instead of just removing the `= null;` and assigning them here.